code/modules/paperwork/photography.dm

```diff
@@ -523,14 +522,19 @@ var/list/SpookyGhosts = list("ghost","shade","shade2","ghost-narsie","horror","s
 	if(..(user, 1))
 		user << "This video camera can send live feeds to the entertainment network. It's [camera ? "" : "in"]active."
 
-
 /obj/item/device/videocam/hear_talk(mob/M as mob, msg)
 	if (camera && on)
 		if(get_dist(src, M) <= canhear_range)
 			talk_into(M, msg)
-		for(var/mob/living/carbon/human/H in watcherslist)
-			H.show_message(text("\blue (Newscaster) [] says, '[]'",M,msg), 1)
+		for(var/obj/machinery/computer/security/telescreen/T)

```

Comment:
If you omit `in ...`, it goes through all of the atoms of that type in the world. This is intentional.